recipes-debian/libpwquality/libpwquality_debian.bb

```diff
@@ -0,0 +1,49 @@
+# base recipe: meta-oe/recipes-extended/libpwquality/libpwquality_1.4.4.bb
+# base branch: master
+# base commit: 57a540a541ca4c3890e0b876b2eaad189a2ed51e
+
+DESCRIPTION = "Library for password quality checking and generating random passwords"
+HOMEPAGE = "https://github.com/libpwquality/libpwquality"
+SECTION = "devel/lib"
+LICENSE = "GPLv2"
```

Comment:
The copyright file in Debian buster says "License: libpwquality or GPL-2+".
https://metadata.ftp-master.debian.org/changelogs//main/libp/libpwquality/libpwquality_1.4.0-3_copyright

Reply:
@hiraku-wfs Thank you! I updated the license field.